src/drive/web/modules/viewer/Footer/BottomSheet.jsx

```diff
@@ -96,6 +103,24 @@ const BottomSheetWrapper = ({
       fullHeight={false}
       onIndexChange={snapIndex => handleOnIndexChange(snapIndex)}
       styles={{ root: styles.root }}
+      threshold={0}
```

Comment:
Should be in config too, no?

Reply:
le `threshold` c'est pour définir une valeur qu'il est possible de dépasser après le point le plus haut. Ca n'a pas trop de sens de mettre autre chose que 0 dans notre cas donc je dirai que non.

De plus, à ce compte là, on pourrait/devrait tout mettre dans la config ? Ce n'est pas très utile à mon sens, et l'idée ici n'est pas de proposer un outil totalement paramétrable (d'ailleurs son comportement par défaut a déjà été validé). L'objectif de cet PR est d'avoir passé le moins de temps possible pour corriger le comportement par défaut qui n'était pas celui attendu.

Reply:
> d'ailleurs son comportement par défaut a déjà été validé 

I guess on Android, but not on iOS, right? This is typically the king of feature where we can have a different config. Let's see if we'll have the same configuration on both platform. I also tend to think it can depend on the smartphone's specifications, so let's have something to play with in order to find the best matches in several context.

> L'objectif de cet PR est d'avoir passé le moins de temps possible pour corriger le comportement par défaut qui n'était pas celui attendu.

Not agreed. Since the beginning of this issue, we talked about making this configuration available to the PO / UX/ Tester in order to give them access to "test, edit, test, edit, test, validate".

Reply:
> De plus, à ce compte là, on pourrait/devrait tout mettre dans la config ?

We should always take a step back every time we develop something and think about what will make it easier to dev / test and validate. If we have something with user's interaction like velocity, friction or else, we have almost 100% chance to put it in config to avoid useless iterations and friction.

Reply:
Hmmm ok je vois. A prendre en compte et à garder en tête pour les prochaines estimations surtout celle dites "quick win"...